Makefile.install

```diff
@@ -20,151 +20,73 @@ ifeq ($(LOCAL),true)
 install:
 	@echo "Nothing to install in a local build!"
 else
-install: install-coq install-coqide install-doc-$(WITHDOC) install-meta
+install: install-coq install-coqide install-doc-$(WITHDOC)
 endif
 
 # NOTA: for install-coqide, see Makefile.ide
 
 install-doc-all: install-doc
 install-doc-no:
+install-byte:
 
-.PHONY: install install-doc-all install-doc-no
+.PHONY: install install-byte install-coqide install-doc-all install-doc-no
 
-#These variables are intended to be set by the caller to make
-#COQINSTALLPREFIX=
-#OLDROOT=
+# These variables are intended to be set by the caller to make, only used
+# by the OSX dmg image so we could get rid of them.
 
-  # Can be changed for a local installation (to make packages).
-  # You must NOT put a "/" at the end (Cygnus for win32 does not like "//").
+# COQINSTALLPREFIX=
+# OLDROOT=
+
+# Can be changed for a local installation (to make packages).
+ # You must NOT put a "/" at the end (Cygnus for win32 does not like "//").
+
+INSTALLLIB:=install -m 644
 
 ifdef COQINSTALLPREFIX
-FULLBINDIR=$(BINDIR:"$(OLDROOT)%="$(COQINSTALLPREFIX)%)
 FULLCOQLIB=$(COQLIBINSTALL:"$(OLDROOT)%="$(COQINSTALLPREFIX)%)
-FULLCONFIGDIR=$(CONFIGDIR:"$(OLDROOT)%="$(COQINSTALLPREFIX)%)
-FULLDATADIR=$(DATADIR:"$(OLDROOT)%="$(COQINSTALLPREFIX)%)
-FULLMANDIR=$(MANDIR:"$(OLDROOT)%="$(COQINSTALLPREFIX)%)
-FULLCOQDOCDIR=$(COQDOCDIR:"$(OLDROOT)%="$(COQINSTALLPREFIX)%)
 FULLDOCDIR=$(DOCDIR:"$(OLDROOT)%="$(COQINSTALLPREFIX)%)
+# Note that substitution here would not work due to make's % pattern not matching the empty string
+FULLCOQPREFIX=$(COQINSTALLPREFIX)
 else
-FULLBINDIR=$(BINDIR)
 FULLCOQLIB=$(COQLIBINSTALL)
-FULLCONFIGDIR=$(CONFIGDIR)
-FULLDATADIR=$(DATADIR)
-FULLMANDIR=$(MANDIR)
-FULLCOQDOCDIR=$(COQDOCDIR)
 FULLDOCDIR=$(DOCDIR)
+FULLCOQPREFIX=$(COQPREFIX)
 endif
```

Comment:
Why did you remove all of these?  At the very least, I think it's wrong to have removed `FULLMANDIR`, as that broke ubuntu packaging, cf https://github.com/coq/coq/issues/14248